react/miniproject/pages/userDashboard.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * This page wiil be used to create dashboard
+ * and use components and actions related to dashboard
+ */
+
+import React, { Component } from 'react';
+import Head from 'next/head';
+import UserDrawer from '../components/userdashboard/userDrawer';
+import UserScoreCard from '../components/userdashboard/userScoreCard';
+import UserMainCard from '../components/userdashboard/userMainCard';
+
+export default class UserDashboard extends Component {
+  render() {
+    return (
+      <div>
+        <Head>
+          <title>MiniProject</title>
+          <meta
+            name="viewport"
+            content="initial-scale=1.0, width=device-width"
+          />
+          <link
+            rel="stylesheet"
+            href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"
+          />
+        </Head>
+        <div>  
+            <UserDrawer/>   
+            <div className="col-sm-12 completeDiv">
+              <UserScoreCard />
+              <UserMainCard/>
+            </div>     
+          <div className="col-sm-12 footer">
```

Comment:
Since footer will be used at several place create a separate component for it and update it in both dashboards